src/Illuminate/Support/Str.php

```diff
@@ -30,6 +30,24 @@ class Str
     protected static $studlyCache = [];
 
     /**
+     * Return the remainder of a string after a given value.
+     *
+     * @param  string  $subject
+     * @param  string  $search
+     * @return string
+     */
+    public static function after($subject, $search)
+    {
+        if (! static::contains($subject, $search)) {
+            return $subject;
+        }
+
+        $searchEndPos = strpos($subject, $search) + static::length($search);
+
+        return static::substr($subject, $searchEndPos, static::length($subject));
```

Comment:
We grab up to the end, so the third parameter should be omitted.
(as more, here it exceeds the end the subject; it works but is erroneous)